sdk/core/core-http/lib/policies/deserializationPolicy.ts

```diff
@@ -153,8 +153,17 @@ export function deserializeResponseBody(jsonContentTypes: string[], xmlContentTy
                       : [];
                   }
                   error.body = operationSpec.serializer.deserialize(defaultResponseBodyMapper, valueToDeserialize, "error.body");
+                  error.parsedBody = error.body;
```

Comment:
Add a comment here as to why we are copying over the body

Reply:
Done